configure.ac

```diff
@@ -895,6 +896,9 @@ if test x$MULTIARCH != x; then
 fi
 AC_SUBST(MULTIARCH_CPPFLAGS)
 
+# checks for UNIX variants that set C preprocessor variables
+AC_USE_SYSTEM_EXTENSIONS
```

Comment:
Can we move this further up, to get rid of the explicit definition of `_GNU_SOURCE` in line 138?